### DISCUSSION TITLE
How to get bright text color for q-(route-)tab?

### DISCUSSION THREAD

**Discussion by eugenevk** [quasarframework/quasar#11938] [General - Components / Directives / etc] [answered]
I am using q-route-tab for my app's navigation menu, but also a q-separator and q-item-label for some actions. It looks like this:

![image](https://user-images.githubusercontent.com/19785727/148533227-a6285d66-2727-4df4-8b48-ee8317500a48.png)

Somehow it looks like the q-route-tab text is not bright white, although I have specified "text-white". How can I get this to bright white, like the separator and the label in between the separators?

This is part of my code:
```
<q-tabs
  active-color="secondary text-white"
  active-bg-color="secondary"
  content-class="bg-primary text-white"
  indicator-color="secondary"
  inline-label
  v-model="tab"
  vertical
>
  <q-route-tab
    exact
    name="timecards"
    no-caps
    :ripple="false"
    style="justify-content:initial"
    :to="{ name: 'Timecards' }"
  >
    <q-icon name="fas fa-business-time" class="q-ml-sm q-mr-md" />Timecards
  </q-route-tab>

  <q-separator color="white" spaced />

  <q-item-label class="text-center text-white text-uppercase">Administration</q-item-label>

  <q-separator color="white" spaced />
...
```

**Comment by pdanpdan** [comment, marked as answer]
Make a codepen so we can play with it

But you should use the inspector and see the `.q-tab--inactive` - adjust it as you want

**Reply by eugenevk** [reply]
Solved by adding this to QTabs:
```
style="opacity: unset"
```
End result looks like this:
```
<q-tabs
	active-color="secondary text-white"
	active-bg-color="secondary"
	content-class="bg-primary text-white"
	indicator-color="secondary"
	inline-label
	style="opacity: unset"
	v-model="tab"
	vertical
>
...
</q-tabs>
```
